DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: algorithmia
 Type: Package
 Title: Allows you to Easily Interact with the Algorithmia Platform
-Version: 0.0.2
-Date: 2016-09-12
+Version: 0.1.0
+Date: 2019-06-11
 Author: James Athappilly
```

Comment:
You should add yourself as an author (don't ask me how to do that though). It looks like the maintainer has to be one email address but I'm not too sure

Reply:
ok will do